Apps/Sandcastle/gallery/iModel Mesh Export Service.html

```diff
@@ -31,10 +31,13 @@
       window.startup = async function (Cesium) {
         "use strict";
         //Sandcastle_Begin
-        const serviceResponse = await fetch("https://api.cesium.com/itwin/token");
-        const { access_token: token } = await serviceResponse.json();
+        // Generate a share key for access to an itwin without oauth
+        // https://developer.bentley.com/apis/access-control-v2/operations/create-itwin-share/
+        Cesium.ITwinPlatform.defaultShareKey =
+          "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpVHdpbklkIjoiNTM1YTI0YTMtOWIyOS00ZTIzLWJiNWQtOWNlZGI1MjRjNzQzIiwiaWQiOiJmZTliOTgyMS0wYWI5LTQ4ZjItYmMzOC01NjQ5MTg5MDQyOTEiLCJleHAiOjE3NDQ5OTA2MjZ9.7bQIX32CGE4slLDPkOQZ4rRr4BqBSjbUOFAUvcdrFXE";
 
-        Cesium.ITwinPlatform.defaultAccessToken = token;
+        // If you do use oauth for user login set:
+        // Cesium.ITwinPlatform.defaultAccessToken = 'your token'
```

Comment:
@jjspace Do we need to have this here? Perhaps an update can point to the tutorial and developer documentation when ready. Same comment for the other sandcastle too.

Reply:
@shehzan10 We can certainly add more docs or tutorials later but I don't think there's an issue with providing this information in multiple locations. These sandcastles may end up being one of the main ways to quick reference how to interact with iTwin data in CesiumJS. I think it's worth keeping these 2 lines just for convenience/reference

Reply:
@jjspace With the goal being to introduce this to the Cesium community via CesiumJS first, IMO its important to keep it simple. By having the comment `If you do use oauth for user login set`, the statement is vauge-ish because OAuth login has a very specific meaning. So either:
1. We remove this comment and instead point to the tutorials/documentation when ready.
2. Make the comment more accurate, yet simple to understand and allows the developer on which approach to use.

Open to other suggestions as well.

Reply:
I agree with @jjspace here, I think its nice to show users that they're able to use either form of authentication to use the sandcastle demo -- sharing or oauth. If you'd like to add more detail to suggesting oauth, you could point them do our oauth docs: https://developer.bentley.com/apis/overview/authorization/

Reply:
Would it be better to then have a comment like:
```
// For alternative forms of authentication you can use, visit https://developer.bentley.com/apis/overview/authorization/.
```

And combine this with line 35?